spec/features/mechanics/add_ride_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
```

Comment:
Since this US is happening on the Show page, I would have just put this in the `spec/features/mechanics/show_spec.rb` file. Not a huge deal